.github/workflows/cd.yml

```
@@ -0,0 +1,124 @@
+  # 処理② Railsの自動デプロイ。動作には ci のクリアが必要
+  deploy-rails:
+    needs: [ci]
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)